### PR TITLE
Add i18n developer documentation

### PR DESCRIPTION
**Pull request by mzazrivec**

### CODE REVIEW

**Comment by chessbyte** [comment]
@Fryguy @matthewd please review

**Comment by Fryguy** [comment]
@mzazrivec Can you also make a PR on ManageIQ proper for the CONTRIBUTING.md file and link to the website location where this will end up (https://manageiq.org/documentation/development/i18n)?

File: i18n.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+Now, commit and push changed files into git (branch, pull request, etc.). Make sure no other files than the following list is commited:
+
+```
+config/model_attributes.rb
+config/locales/manageiq.pot
+config/locales/*/*.po
+```
+
+* Upload the catalog created in previous step to [zanata](http://translate.zanata.org). You need to be maintainer of ManageIQ project at [zanata](http://translate.zanata.org)
```
Comment (Fryguy) [inline i18n.md]: @mzazrivec Note that I am working on making it so we don't have to make every contributor a maintainer.  The next version of zanata will have Team-specific languages groups, and we can make people within those groups maintainers, reviewers, or just translators.

**Comment by mzazrivec** [comment]
@Fryguy Something like pr #6298 ?

**Pull request merged**